File: 2002B2/VB/WinForms-Framework-Implements-LoggingSample/WinLogger/Logging/ILog.vb
Imports System.Diagnostics
'******************************************************
'* Description: This is an interface which
'*              has one abstract method - WriteLog.
'*              Components that support this interface 
'*              will provide the implementation of this method.
'******************************************************
    Interface ILog
        Sub WriteLog(ByVal strMsg As String, ByVal eLogType As EventLogEntryType)
    End Interface

File: 2002B2/CS/WinForms-Framework-Implements-LoggingSample/WinLogger/Logging/ILog.cs
using System.Diagnostics;

//******************************************************
//* Description: This is an interface which
//*              has one abstract method - WriteLog.
//*              Components that support this interface 
//*              will provide the implementation of this method.
//******************************************************
public interface ILog
{
	void WriteLog(string strMsg, System.Diagnostics.EventLogEntryType eLogType);
}

